### DISCUSSION TITLE
How to pass value to q-input?

### DISCUSSION THREAD

**Discussion by fairking** [quasarframework/quasar#16717] [General - Components / Directives / etc] [answered]
Hi everyone,

I am struggling with very simple task which usually works in any other vuejs components (other frameworks).
How to pass a single static value to the input?

The following example not working:
``` html
<q-input
         value="Hello world!"
         readonly />
```

https://jsfiddle.net/31owakzb/

**Comment by pdanpdan** [comment, marked as answer]
The same way you do it in any vue3 situation, using model-value
